cmd/init.go

```diff
@@ -100,7 +100,7 @@ var initCmd = &cobra.Command{
 		initOptions.Verbose = verbose
 		initOptions.BlockchainProvider, _ = stacks.BlockchainProviderFromString(blockchainProviderInput)
 		initOptions.DatabaseSelection, _ = stacks.DatabaseSelectionFromString(databaseSelection)
-		initOptions.TokensProvider, _ = stacks.TokensProviderFromString(tokensProviderSelection)
+		initOptions.TokenProviders, _ = stacks.TokensProviderFromStrings(tokenProvidersSelection)
```

Comment:
nit: "TokensProviderFromStrings" is now inconsistent with everything that was renamed "TokenProviders"